src/library/f3d_window_impl_standard.cxx

```diff
@@ -98,6 +101,10 @@ window_impl_standard::window_impl_standard(const options& options, bool offscree
 {
   this->Internals->RenWin->SetMultiSamples(0); // Disable hardware antialiasing
   this->Internals->RenWin->SetOffScreenRendering(offscreen);
+
+  // set default window name / icon
+  this->setWindowName(f3d::AppTitle);
```

Comment:
@mwestphal setting the title and icon here produce a segfault. Is it related to the recent fix you did recently in VTK?

Reply:
I dont see the relation to the fix.

Reply:
what is the backtrace ?

Reply:
```gdb
#0  0x00007ffff5db0940 in XInternAtom () from /usr/lib/libX11.so.6
#1  0x00007ffff6234957 in vtkXOpenGLRenderWindow::SetIcon (
    this=0x5555555c45b0, img=0x5555555ad3e0)
    at /home/michael/dev/vtk/Rendering/OpenGL2/vtkXOpenGLRenderWindow.cxx:1542
#2  0x00007ffff7ed7220 in f3d::window_impl_standard::setIcon (
    this=0x5555555daf80, icon=0x7ffff7f21260 <F3DIcon>, iconSize=2487)
    at /home/michael/dev/f3d/src/library/f3d_window_impl_standard.cxx:121
#3  0x00007ffff7ed70f7 in f3d::window_impl_standard::window_impl_standard (
    this=0x5555555daf80, options=..., offscreen=false)
    at /home/michael/dev/f3d/src/library/f3d_window_impl_standard.cxx:107
#4  0x00007ffff7ebdff0 in std::make_unique<f3d::window_impl_standard, f3d::options&, bool> () at /usr/include/c++/11.2.0/bits/unique_ptr.h:962
#5  0x00007ffff7ebc1f6 in f3d::engine::getWindow (this=0x5555555db040)
    at /home/michael/dev/f3d/src/library/f3d_engine.cxx:60
#6  0x00007ffff7ebc297 in f3d::engine::getLoader (this=0x5555555db040)
    at /home/michael/dev/f3d/src/library/f3d_engine.cxx:74
#7  0x00007ffff7ebc361 in f3d::engine::getInteractor (this=0x5555555db040)
    at /home/michael/dev/f3d/src/library/f3d_engine.cxx:85
#8  0x000055555555dcb3 in F3DStarter::Start (this=0x7fffffffdf70, argc=3, 
    argv=0x7fffffffe0b8)
    at /home/michael/dev/f3d/src/application/F3DStarter.cxx:75
#9  0x000055555555d492 in main (argc=3, argv=0x7fffffffe0b8)
    at /home/michael/dev/f3d/src/application/main.cxx:13
```